ocaml/testsuite/tests/parse-errors/optional_tuple_component2.compilers.reference

```diff
@@ -1,4 +1,4 @@
-File "optional_tuple_component2.ml", line 8, characters 9-10:
-8 | ~~(y, ?x:2)
-             ^
-Error: Tuple components cannot be optional
+File "optional_tuple_component2.ml", line 8, characters 4-7:
```

Comment:
I think these tests can just go, now that there isn't any special error to test.